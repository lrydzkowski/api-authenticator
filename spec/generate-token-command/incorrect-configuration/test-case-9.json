{
  "app - dev": {
    "clientId": "client_id",
    "scope": "openid offline_access",
    "tokenEndpoint": "https://login.microsoftonline.com/{tenant_id}/oauth2/token",
    "flow": "client_credentials"
  }
}
