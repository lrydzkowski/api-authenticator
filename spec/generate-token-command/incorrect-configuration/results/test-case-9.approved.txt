The given configuration is incorrect. Validation errors: '{"clientSecret":"You have to add 'clientSecret' property to the configuration."}'.
